driver/src/main/java/eu/cloudnetservice/driver/service/ServiceDeployment.java

```diff
@@ -171,6 +186,38 @@ public static class Builder {
       return this;
     }
 
+    /**
+     * Sets the file names (in a regular expression form) which should be included when deploying a service. Directories
+     * must be suffixed with a {@literal /}.
+     * <p>
+     * The given includes collection will get copied into the builder, meaning that further modification of it will not
+     * reflect into the builder and vice-versa.
+     *
+     * @param includes the includes of the deployment.
+     * @return the same instance as used to call the method, for chaining.
+     * @throws NullPointerException if then given includes collection is null.
+     */
+    public @NonNull Builder includes(@NonNull Collection<String> includes) {
+      this.includes = new HashSet<>(includes);
+      return this;
+    }
+
+    /**
+     * Adds the file names (in a regular expression form) which should be included when deploying a service. Directories
+     * must be suffixed with a {@literal /}.
+     * <p>
```

Comment:
We should include a paragraph here and on the `includes` method that explains that when exclusions are added to the deployment that excludes have no effect anymore, or preferability we should honor includes and excludes when provided 🤔